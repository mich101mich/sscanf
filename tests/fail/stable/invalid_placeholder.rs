fn main() {
    sscanf::sscanf!("", "{", str);
    sscanf::sscanf!("", "{str");
    sscanf::sscanf!("", "{:", str);
    sscanf::sscanf!("", "{:b", str);
    sscanf::sscanf!("", "{://", str);
    sscanf::sscanf!("", "{:}}", str);
    sscanf::sscanf!("", "}", str);
    sscanf::sscanf!("", ":}", str);
    sscanf::sscanf!("", "{{:}", str);
    sscanf::sscanf!("", "{/.*?/}", str);
}
