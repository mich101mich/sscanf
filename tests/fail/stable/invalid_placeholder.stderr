error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{':
       At "{"
           ^
 --> tests/fail/stable/invalid_placeholder.rs:2:25
  |
2 |     sscanf::sscanf!("", "{", str);
  |                         ^^^

error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{':
       At "{str"
           ^^^^
 --> tests/fail/stable/invalid_placeholder.rs:3:25
  |
3 |     sscanf::sscanf!("", "{str");
  |                         ^^^^^^

error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{':
       At "{:"
           ^^
 --> tests/fail/stable/invalid_placeholder.rs:4:25
  |
4 |     sscanf::sscanf!("", "{:", str);
  |                         ^^^^

error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{':
       At "{:b"
           ^^^
 --> tests/fail/stable/invalid_placeholder.rs:5:25
  |
5 |     sscanf::sscanf!("", "{:b", str);
  |                         ^^^^^

error: missing '}' to close a placeholder. If the '{' was intended to be a literal, escape it with '{{':
       At "{://"
           ^^^^
 --> tests/fail/stable/invalid_placeholder.rs:6:25
  |
6 |     sscanf::sscanf!("", "{://", str);
  |                         ^^^^^^

error: format options cannot be empty. Consider removing the ':':
       At "{:}}"
             ^
 --> tests/fail/stable/invalid_placeholder.rs:7:25
  |
7 |     sscanf::sscanf!("", "{:}}", str);
  |                         ^^^^^^

error: unexpected standalone '}'. Literal '}' need to be escaped as '}}':
       At "}"
           ^
 --> tests/fail/stable/invalid_placeholder.rs:8:25
  |
8 |     sscanf::sscanf!("", "}", str);
  |                         ^^^

error: unexpected standalone '}'. Literal '}' need to be escaped as '}}':
       At ":}"
            ^
 --> tests/fail/stable/invalid_placeholder.rs:9:25
  |
9 |     sscanf::sscanf!("", ":}", str);
  |                         ^^^^

error: unexpected standalone '}'. Literal '}' need to be escaped as '}}':
       At "{{:}"
              ^
  --> tests/fail/stable/invalid_placeholder.rs:10:25
   |
10 |     sscanf::sscanf!("", "{{:}", str);
   |                         ^^^^^^

error: missing `:` in front of custom regex. Write `{:/.*?/}` instead:
       At "{/.*?/}"
            ^^^^^
  --> tests/fail/stable/invalid_placeholder.rs:11:25
   |
11 |     sscanf::sscanf!("", "{/.*?/}", str);
   |                         ^^^^^^^^^
